packages/theming/apple-theme/src/appleHighContrast.macos.ts

```diff
@@ -0,0 +1,5 @@
+export let isHighContrast = false;
+
+export function setIsHighContrast(isHighContrast: boolean) {
```

Comment:
nit: I think if we have a set function we should also make a get function. Also makes the code more encapsulated.